instrumentation/kafka-streams/README.md

```diff
@@ -1,22 +1,59 @@
 # Brave Kafka Streams instrumentation
 
-Add decorators for Kafka Streams client to enable tracing.
-* `TracingKafkaClientSupplier` a client supplier which traces send and receive operations.
+Add decorators for Kafka Streams to enable tracing.
+* `TracingKafkaClientSupplier` a client supplier which traces poll and send operations.
+* `TracingProcessorSupplier` completes a span on `process`
+* `TracingTransformerSupplier` completes a span on `transform`
 
 ## Setup
-First, setup [our Kafka Client instrumentation](../kafka-clients/README.md) like this:
+First, setup the generic Kafka Streams component like this:
 ```java
-kafkaTracing = KafkaTracing.newBuilder(tracing)
-                           .remoteServiceName("my-broker")
-                           .build();
+kafkaStreamsTracing = KafkaStreamsTracing.create(kafkaTracing);
 ```
 
-Next, create a streams component like this:
+To add a Tracing Processor to your application use the `TracingProcessorSupplier` provided by instrumentation:
 ```java
-kafkaStreamsTracing = KafkaStreamsTracing.create(kafkaTracing);
+builder.stream(inputTopic)
+       .processor(kafkaStreamsTracing.processorSupplier(
```

Comment:
sometimes I use same name as library. ex if they say "transform" I do too. ex in sparkjava we do this

Reply:
In this case we are using the same name as the parameter: e.g. `.processor(ProcessorSupplier)` -> `.processor(kafkaStreamsTracing.processorSupplier(...))` 
Do you think naming it the same as the operation will be more readable? e.g. `.processor(ProcessorSupplier)` -> `.processor(kafkaStreamsTracing.processor(...))`